android-agent/src/test/java/io/opentelemetry/android/OpenTelemetryRumBuilderTest.java

```diff
@@ -308,6 +309,17 @@ void verifyGlobalAttrsForLogs() {
                                 .build());
     }
 
+    @Test
+    void verifyServicesAreInitialized() {
+        makeBuilder().build();
+
+        try {
+            ServiceManager.get();
+        } catch (IllegalStateException e) {
+            fail("Services were not initialized.");
+        }
```

Comment:
Prefer to just let the exception leave the method and fail the test. I might also add a non-null on the result of `get()`.

Reply:
Thanks, it's updated now.